### PR TITLE
Add telescope page

### PR DESCRIPTION
**Pull request by mcoughlin**
The final PR in the trio to add a Telescope page to the front end.

### CODE REVIEW

**Comment by mcoughlin** [comment]
@guynir42 Maybe we get this one in next and then do the allocations page?

File: config.yaml.defaults [hunk 1]
```diff
@@ -132,6 +134,10 @@ app:
       icon: Info
       url: /about
 
+    - name: Telescopes
+      icon: Toys
+      url: /telescopes
+
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply config.yaml.defaults]: Ok MyLocation for now... can always switch when we figure out how to do something fancier? I moved Telescope up top.

File: static/js/components/TelescopePage.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+  let result = "";
+
+  if (telescope?.lat) {
+    result += "( ";
+    if (telescope?.lat) {
+      result += `/ Latitude: ${telescope.lat}`;
+    }
+    if (telescope?.lon) {
+      result += `/ Latitude: ${telescope.lon}`;
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply static/js/components/TelescopePage.jsx]: good catch.

File: static/js/components/TelescopePage.jsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      </div>
+    );
+  }
+
+  let result = "";
+
+  if (telescope?.lat) {
+    result += "( ";
+    if (telescope?.lat) {
+      result += `/ Latitude: ${telescope.lat}`;
```
[the remark this answers is not in the document]
Comment (mcoughlin) [reply static/js/components/TelescopePage.jsx:60]: I wasn't sure especially with the ? and my lack of knowing anything about JS.

[the remark this answers is not in the document]
Comment (guynir42) [reply static/js/components/TelescopePage.jsx:60]: Try to conditionally put them into an array and then use join()

[the remark this answers is not in the document]
Comment (mcoughlin) [reply static/js/components/TelescopePage.jsx]: yes that is cleaner.

File: static/js/components/TelescopePage.jsx [hunk 4]
```diff
@@ -52,21 +51,14 @@ export function telescopeInfo(telescope) {
     );
   }
 
-  let result = "";
+  const array = [
+    ...(telescope?.lat ? [`Latitude: ${telescope.lat}`] : []),
+    ...(telescope?.lat ? [`Longitude: ${telescope.lon}`] : []),
```
Comment (guynir42) [inline static/js/components/TelescopePage.jsx:56]: you're using `.lat` three times...

**Review by guynir42 — APPROVED** [review]
If the tests look OK then you can merge.

**Comment by mcoughlin** [comment]
Failure unrelated. Merging.

**Comment by stefanv** [comment]
:pray: this is a very clean and tidy PR; thank you @mcoughlin!

I'm sorry to be a pain, but please *please* in the future add proper commit messages.  Fine to put it in the PR description, then we can use it from there:

```
* Add telescope page

* note flaky!

* Guy comments

* Guy comment

* bad bad bad
```

@guynir42 Don't hesitate to edit commit messages of PRs, especially when they combine several commits, before merging.

From our developer documentation at docs.fritz.science:

> Each commit message should consist of a summary line and a longer description, e.g.:
>
> Rewrite the contributor guide
> 
> While reading through the contributor guide, I noticed several places
in which instructions were out of order. I therefore reorganized all
sections to follow logically, and fixed several grammar mistakes along
the way.

**Comment by guynir42** [comment]
@stefanv will do! I didn't notice I was able to edit these...

**Pull request merged**